share/backend_config.ini

```diff
@@ -17,6 +17,12 @@ max_zonemaster_execution_time            = 300
 number_of_processes_for_frontend_testing = 20
 number_of_processes_for_batch_testing    = 20
 
+#
+# WARNING: This option is experimental and all edge cases are not fully tested, don't use, or use with care.
+# This option allow to customize the number of retries of a test if it is not finished aftert max_zonemaster_execution_time seconds.
+#
+#maximal_number_of_retries=3
```

Comment:
Since the default number is 0, the value should be 0.

Better language, I think (and better with shorter lines):
```
# WARNING: This option is experimental and all edge cases are not fully tested.
# Do not use it (keep the default value "0"), or use it with care.
```

Reply:
I don't think the suggested value for this option should be 0 in the .ini file.
As the default values is 0 (in the code) activating this option and putting there a value of 0 makes no sense.
The warning also applies only for values other than 0.

Reply:
In my experience it's conventional to keep the default value commented out. But since you make the effort to state the default value explicitly I don't think it's very important.